api/tests/unit/segments/test_unit_segments_audit_helpers.py

```
@@ -0,0 +1,251 @@
+
+    # Then
+    assert result == [
+        {
+            "type": "ALL",
+            "conditions": [{"operator": "PERCENTAGE_SPLIT", "value": "50"}],
+        }
+    ]
+
+
+def test_get_segment_rules_change_details__rules_changed__returns_diff(
+    project: "Project",  # type: ignore[name-defined]
```

Ruff (0.15.21)

[error] Undefined name `Project`

(F821)